frontend/src/components/eam/components/tenant/tenant.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from 'react';
+import { Button, Input } from 'components/common/common';
+import { InputType, ButtonType } from 'common/enums/enums';
+import {
+  useAppForm,
+  useAppDispatch,
+  useAppSelector,
+  useEffect,
+  useMemo,
+} from 'hooks/hooks';
+import { getNameOf } from 'helpers/helpers';
+import {
+  EAMTenantUpdateRequestDto,
+  EAMTenantFormDto,
+  EAMTenantByIdResponseDto,
+} from 'common/types/types';
+import { eamTenantUpdate as EAMTenantValidationSchema } from 'validation-schemas/validation-schemas';
+import { eam as EAMTenantConfigurateActions } from 'store/actions';
+import { mapTenantDefaultValues } from './helpers/helpers';
+import styles from './styles.module.scss';
+
+const Tenant: FC = () => {
+  const dispatch = useAppDispatch();
+  const { tenant } = useAppSelector(({ app }) => ({
+    tenant: app.tenant,
+  }));
+
+  const defaultValues = useMemo(
+    () => mapTenantDefaultValues(tenant as EAMTenantByIdResponseDto),
```

Comment:
```suggestion
    () => mapTenantDefaultValues(tenant),
```
we discussed that we should cast a type